stories/Themes/index.stories.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import * as Themes from '../../src/Themes';
+import {createStoryTitle} from "../Constants";
 
-const stories = storiesOf('Theme', module).addDecorator(withKnobs);
+const stories = storiesOf(createStoryTitle('Theme'), module).addDecorator(withKnobs);
 Object.entries(Themes).forEach(([key, value]) => {
     stories.add(`with ${key}`, () => <span>To be automated</span>);
 });
```

Comment:
This story's purpose is to show light or dark theme? With the additon of Design system colors this seems redundant, but maybe it will be used to display components?

Reply:
@fpunny what is this file used for? Can't we just use Storybook Globals to change the theme?